### PR TITLE
revise synonyms for MONDO_0005550

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
addresses #750

### CODE REVIEW

**Comment by nicolevasilevsky** [comment]
@cmungall see question on ticket #750 (pasted here: in the current PR, I annotated "ID" as an abbreviation and as 'synonym to be removed from public release but maintained in edit version as record of external source' - does that properly address the issue?)

File: .travis.yml [hunk 1]
```diff
@@ -25,13 +25,14 @@ before_install:
...
      ]}'
-  - docker pull obolibrary/odkfull
+  #- docker pull obolibrary/odkfull
 
 before_script:
   - cd src/ontology
 
 script:
-  - ./run.sh make test
+  - echo "SKIPPING TRAVIS REAL BUILD TO FORCE SUCCESS"
```
Comment (cmungall) [inline .travis.yml]: we shouldn't be skipping travis!

File: .travis.yml [hunk 2]
```diff
@@ -25,13 +25,14 @@ before_install:
        {"name":"TRAVIS_BUILD_WEB_URL", "value":"'"${TRAVIS_BUILD_WEB_URL}"'"},
        {"name":"TRAVIS_BUILD_NUMBER", "value":"'"${TRAVIS_BUILD_NUMBER}"'"}
      ]}'
-  - docker pull obolibrary/odkfull
+  #- docker pull obolibrary/odkfull
```
Comment (cmungall) [inline .travis.yml]: ?

**Comment by cmungall** [comment]
Can you undo your travis changes?

File: .travis.yml (hunk 2 above)
Comment (nicolevasilevsky) [reply .travis.yml]: @ShahimEssaid

File: .travis.yml (hunk 1 above)
Comment (nicolevasilevsky) [reply .travis.yml]: @ShahimEssaid

**Comment by ShahimEssaid** [comment]
@cmungall  can you see #743 and then I'll undo it if you still think it is a bad idea. Undoing it will start to give overall "failed" statuses and one will need to look further to see if the Jenkins status succeeded. At this point the Travis status is effectively irrelevant since it is almost always timing out.

**Comment by matentzn** [comment]
I think if we are a bit economical we can squeeze the duration of the overall pipeline on travis down to 40 min, well within the acceptable range. Many of the sparql tests are simply redundant!

**Comment by nicolevasilevsky** [comment]
pinging @cmungall and @ShahimEssaid - I think the next step are dependent on you two :)

**Comment by cmungall** [comment]
Closing this. Make a new PR for the synonym change. Make a new change for travis if required.

**Pull request closed without merging**